src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/FixAllContext.cs

```diff
@@ -207,11 +202,6 @@ private async Task<ImmutableArray<Diagnostic>> GetProjectDiagnosticsAsync(Projec
         {
             Contract.ThrowIfNull(project);
 
-            if (this.Project.Language != project.Language)
-            {
-                return ImmutableArray<Diagnostic>.Empty;
-            }
```

Comment:
this will not work.  `DiagnosticProvider`s are often language specific.  So most will not be able to process diagnostics from another language.  Worse, they will legit just crash if given the wrong diagnostic language as they all assume they'll only get diagnostics from teh same language.